srcpkgs/obfs4proxy/template

```diff
@@ -11,8 +12,8 @@ short_desc="Pluggable transport proxy for Tor, implementing obfs4"
 maintainer="Daniel Eyßer <daniel.eysser@gmail.com>"
 license="BSD-2-Clause"
 homepage="https://gitlab.com/yawning/obfs4"
-distfiles="https://people.torproject.org/~yawning/releases/obfs4proxy/obfs4proxy-${version}.tar.xz"
```

Comment:
https://gitweb.torproject.org/pluggable-transports/obfs4.git/

Reply:
didn't know it's a development version :/ should we revert?